src/utils/money.cljs

```diff
@@ -285,10 +285,14 @@
   [bn1 bn2]
   (.round (.mul ^js bn1 bn2) 0))
 
-(defn div
+(defn- div*
   [bn1 bn2]
   (.dividedBy ^js bn1 bn2))
 
+(def div
+  "Divides with defaults, this version is able to receive `nil` and takes them as 0."
+  (fnil div* (bignumber 0) (bignumber 1)))
```

Comment:
Added `fnil` wrapper with 0 and 1 values produces 0 (because 0 divided by 0 will provide **NaN**).